.gitlab-ci.yml

```diff
@@ -629,10 +608,15 @@ test_backend_integration:
     variables:
       - $RUN_INTEGRATION_TESTS == "true"
 
-.template_test_integ: &test_integration
+test_integration:
```

Comment:
I am tempted to call this job `test_client_integration`, or `test_system_integration"... I've always been a bit uncomfortable with "integration" vs "backend integration"

Reply:
How about "full integration"?

"System integration" also works, IMHO.